src/interface/data_collection.hpp

```diff
@@ -76,17 +70,21 @@ class DataCollection {
     containers_[key] = c;
     return containers_[key];
   }
+
   template <class SRC_t, typename ID_t = std::string>
   std::shared_ptr<T> &Add(const std::string &label, const std::shared_ptr<SRC_t> &src,
                           const std::vector<ID_t> &fields = {}) {
     return Add(label, src, fields, false);
   }
+
   template <class SRC_t, typename ID_t = std::string>
   std::shared_ptr<T> &AddShallow(const std::string &label,
                                  const std::shared_ptr<SRC_t> &src,
                                  const std::vector<ID_t> &fields = {}) {
     return Add(label, src, fields, true);
   }
+  // TODO(LFR): Is this method used anywhere? Should insertion of empty containers be
+  // allowed?
   std::shared_ptr<T> &Add(const std::string &label);
```

Comment:
Is `base` created this way?

Reply:
If I remove the method, Parthenon itself compiles with no issues. I just wonder if any downstream codes use this. It doesn't really make sense when `T = MeshData` though, since the label is not the key in that case.

I am removing it, since I don't see any use case and it seems like it could result in unexpected behavior if it is used.